lib/ReactViews/Map/Panels/SharePanel/ShareUrl/ShareUrlInput.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from "react";
+
+import Input from "../../../../../Styled/Input";
+import { useTranslation } from "react-i18next";
+
+interface IStoryShareInput {
+  theme?: "light" | "dark";
+  shareUrl: string;
+  placeholder?: string;
+}
+
+export const ShareUrlInput: FC<IStoryShareInput> = ({
```

Comment:
This could almost certainly be added into `ShareUrlClipboard`. Would we ever use `ShareUrlInput` in another component do you think?